tools/train.py

```diff
@@ -150,8 +186,7 @@ def main():
         # save mmdet version, config file content and class names in
         # checkpoints as meta data
         cfg.checkpoint_config.meta = dict(
-            mmdet_version=__version__,
-            config=cfg.pretty_text,
+            mmdet_version=__version__ + get_git_hash()[:7],
```

Comment:
This will only get the git hash of mmdet3d rather than mmdet.

Reply:
So we only need to get the version of mmdet? Is it correct to remove the get_git_hash() because the get_git_hash() only returns the git hash of the current repo of mmdet3d?